portfolio/src/main/webapp/index.html

```diff
@@ -10,14 +10,60 @@
       <!-- After added content and css styling report back to step 11 -->
       <!-- to access the walkthrough: teachme ~/software-product-sprint/walkthroughs/week-1-web-development/portfolio-walkthrough.md -->
     <div id="content">
-      <img class="main-image" src="images/LinkedIn.png"/>
+      <img class="image" src="images/LinkedIn.png"/>
       <h1>Samantha's Portfolio</h1>
       <p>This is my portfolio.</p>
       <p>Hello! My name is Samantha Murray Tuesta and I am a second-year undergraduate student at Western Washington University with a passion for all things computer science! Welcome to my portfolio!</p>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to learn a fun fact about me:</p>
       <button onclick="addFunFact()">Fact</button>
       <div id="greeting-container"></div>
+      <img class="profile-pic" src="images/profilePicture.JPG"/>      
+      <h3>About Me</h3>
+      <p>I am a first generation college student, in my second year studying Computer Science and Internet Studies. I am currently based in Seattle, WA, and attend college in Bellingham!
+         <br><br>At my institution, Western Washington University, I am engaged in research specializing in digital privacy and its portrayal in the media. My work is aimed at creating a digital web game to engage students of all ages and teach the importance of digital privacy and how we go about advocating for those impacted by breaches in that privacy.
+         <br><br>Some of my favorite activites include playing piano and cello in my free time, reading, drawing, and designing digital web pages! I first fell in love with computer science through learning HTML and enjoy incorporating that love in my research and studies!
+         <br><br>Growing up, I often visited my mother's hometown of Chiclayo, Peru and consider it my second home. I have always grown up with a love of culture and hope to one day explore other cultures to broaden my world-view!  
+      </p>
+      <h3>Projects</h3>
+      <ul>
+          <li>
```

Comment:
This seems like a bit of an abuse of the intent of the `<ul>` tag to me. We can discuss later.

Reply:
Yes I agree! I have changed this to normal paragraph blocks, and incorporated the use of the ul tag in listing off my socials!